torchtune/data/_messages.py

```diff
@@ -161,20 +161,23 @@ class InputOutputToMessages(Transform):
             keeping the default "input" and "output" column names.
         new_system_prompt (Optional[str]): if specified, prepend a system message. This can
             serve as instructions to guide the model response. Default is None.
+        if_multimodal (bool): Whether is dataset multimodal or not.
 
     Raises:
         ValueError: If ``column_map`` is provided and ``input`` not in ``column_map``, or
-            ``output`` not in ``column_map``.
+            ``output`` not in ``column_map`` or ``image`` not in ``column_map``.
     """
 
     def __init__(
         self,
         train_on_input: bool = False,
         column_map: Optional[Dict[str, str]] = None,
         new_system_prompt: Optional[str] = None,
+        is_multimodal: bool = False,
```

Comment:
can we infer this similar to in ShareGPTToMessages?
```
is_multimodal = "image" in sample or (
            "image" in self._column_map and self._column_map["image"] in sample
        )
```
so users don't need to worry about another flag

Reply:
Sure!